wg-charter.html

```diff
@@ -0,0 +1,387 @@
+        <h2>
+          Deliverables
+        </h2>
+
+        <div id="normative">
+          <h3>
+            Normative
+          </h3>
+          <dl>
+          <dt>WebGPU API</dt>
+
+          <dd>
+              An API as described by the <a
+              href="https://www.w3.org/community/gpu/">GPU for the Web
+              Community Group</a> charter.
+              <p class="milestone"><b>Recommendation expected completion:</b> Q2 2020</p>
+          </dd>
+
+          <dt>WebGPU Test Suite</dt>
+
+          <dd>
+              A conformance test suite for the WebGPU API.
+              <p class="milestone"><b>Expected delivery:</b> Q2 2020</p>
+          </dd>
+
+          </dl>
+
+          <h3>
+            Optional Specifications
+          </h3>
+          <dl>
+          <dt>WebGPU Shading Language</dt>
+
+          <dd>
+              A Shading Language specification that defines the programmable
+              interface to the GPU for the Web graphics and computation
+              pipeline. Alternatively, the Working Group may recommend
+              an existing Shading Language.
+          </dd>
+          </dl>
+        </div>
+
+        <div id="wg-other-deliverables">
+            <h3>
+              Other deliverables
+            </h3>
+            <p>
+                The GPU for the Web Community group will produce deliverables to be
+                used within this Working Group. In particular, a conformance test suite
+                and possibly reference implementations.
+            </p>
+        </div>
+      </section>
+
+
+      <section id="coordination">
+        <h2>Coordination</h2>
+        <p>For all specifications, this Working Group will seek <a href="https://www.w3.org/Guide/Charter.html#horizontal-review">horizontal review</a> for accessibility, internationalization, performance, privacy, and security with the relevant Working and Interest Groups, and with the <a href="https://www.w3.org/2001/tag/" title="Technical Architecture Group">TAG</a>. Invitation for review must be issued during each major standards-track document transition, including <a href="https://www.w3.org/2018/Process-20180201/#RecsWD" title="First Public Working Draft">FPWD</a> and <a href="https://www.w3.org/2018/Process-20180201/#RecsCR" title="Candidate Recommendation">CR</a>, and should be issued when major changes occur in a specification.</p>
+
+        <p>Additional technical coordination with the following Groups will be made, per the <a href="https://www.w3.org/2018/Process-20180201/#WGCharter">W3C Process Document</a>:</p>
+
+        <div>
+          <h3 id="w3c-coordination">W3C Groups</h3>
+          <dl>
+            <dt><a href="https://www.w3.org/2001/tag/">Technical Architecture Group (TAG)</a></dt>
+        <dd>The WG may ask the Technical Architecture Group to review some of its specifications.</dd>
+
+        <dt><a href="https://www.w3.org/WebPlatform/WG/">Web Platform Working Group</a></dt>
```

Comment:
This group is basically defunct (it's a zombie group still somehow living after its charter expired 🙂). You want to link to [Web Applications Working Group](https://www.w3.org/2019/webapps/) instead. The new mission text is: "The [...] Web Applications Working Group (WebApps WG) [... produces] specifications that facilitate the development of client-side web applications."